Source/santasyncservice/SNTSyncEventUpload.mm

```diff
@@ -87,41 +87,49 @@ - (BOOL)performRequest:(::pbv1::EventUploadRequest *)req {
 
 - (BOOL)uploadEvents:(NSArray<SNTStoredEvent *> *)events {
   google::protobuf::Arena arena;
+  google::protobuf::Arena *pArena = &arena;
   NSMutableSet *eventIds = [NSMutableSet setWithCapacity:events.count];
   auto req = google::protobuf::Arena::Create<::pbv1::EventUploadRequest>(&arena);
   req->set_machine_id(NSStringToUTF8String(self.syncState.machineID));
   google::protobuf::RepeatedPtrField<::pbv1::Event> *uploadEvents = req->mutable_events();
+  __block BOOL success = YES;
+  NSUInteger finalIdx = (events.count - 1);
 
-  NSUInteger i = 0;
-  while (i < events.count) {
-    SNTStoredEvent *event = events[i++];
-
+  [events enumerateObjectsUsingBlock:^(SNTStoredEvent * event, NSUInteger idx, BOOL *stop){
```

Comment:
```suggestion
  [events enumerateObjectsUsingBlock:^(SNTStoredEvent * event, NSUInteger idx, BOOL *stop) {
```

Reply:
Had a couple issues on this line, I ran the linter.